descartes_planner/test/dense_planner.cpp

```diff
@@ -37,7 +39,7 @@ class ThreeDOFRobot: public descartes_trajectory_test::CartesianRobot
 {
 public:
   ThreeDOFRobot():
-    descartes_trajectory_test::CartesianRobot(0,0,3)
+    descartes_trajectory_test::CartesianRobot(0,0)
```

Comment:
Should the name of this class, `ThreeDOFRobot` be changed?  Doesn't this create a 6DOF robot?